scipy/interpolate/tests/test_polyint.py

```diff
@@ -370,7 +371,7 @@ def test_overshoot(self):
                 y1, y2 = y2, y1
             xp = np.linspace(x1, x2, 10)
             yp = p(xp)
-            assert_(((y1 <= yp) & (yp <= y2)).all())
+            assert_(((y1 <= yp + 1e-15) & (yp <= y2 + 1e-15)).all())
```

Comment:
There is some deviation on the scale of machine epsilon (~1e-16) on a single segment.